README.md

```diff
@@ -0,0 +1,5 @@
+# FCC Test Suite Template
+
+**Fork** this project (do not open a Pull Request) to start working on any of freeCodeCamp's responsive web design course projects. Select the project from the dropdown menu and run the tests to see which requirements your project is still missing.
+
+You don't know how to fork a project? [Click hear](https://help.github.com/articles/fork-a-repo/) to learn how to do it.
```

Comment:
I think you can improve your README file. Please 🙏🏽 follow the instructions from the [README template](https://github.com/microverseinc/readme-template/blob/master/README.md). Kindly add a meaningful title to your pull request.

If you don't have yet the knowledge to complete some info into the template leave it blank and fill it out when you have it.

:reminder_ribbon: Remember to recognize the contribution of BABEL and freecodecamp guys by the CSS/js you used in your project.